packages/backend/src/open_payments/payment/outgoing/service.ts

```diff
@@ -257,7 +256,8 @@ async function validateGrant(
   deps: ServiceDependencies,
   payment: OutgoingPayment,
   grant: Grant,
-  callback?: (f: unknown) => NodeJS.Timeout
+  callback?: (f: unknown) => NodeJS.Timeout,
+  grantLockTimeoutMs: number = 5000
```

Comment:
allow setting this locking value so we can set it to something much lower in tests